### ISSUE TITLE
Upperbody frankmocap only and too much shaking? 

### ISSUE THREAD

**Issue by krigeta** [facebookresearch/frankmocap#134]
Hello I am using it for upper body but the problem is that when i use fullbody commands it works kinds okay but when i try to use the frankmocap body the things goes wild, why it is happening?

Hello, I'm using it to capture the upper body, but the problem is that it works fine when I use **body motion capture** commands, but when I try to use the **whole body motion capture**, strange things happen(the shake is too much and it is thinking that my mic is a person)

**Comment by penincillin** [comment]
@krigeta 
Sorry, I am little confused. Under which scenario the capturing results are not good, is it body-motion or full-body-motion ?

**Comment by krigeta** [comment]
Full body motion but I am inputting a video which is showing only upper body. even I try this same video both of the body-motion or full-body-motion and for some reason full-body motion is detecting it with very less accuracy

**Comment by penincillin** [comment]
Get it. 
It is possible that our algorithm generates unstable predictions when only upperbody visible, since most of the training data have full-body visible and we do no specific process for the occlusion scenarios.

**Comment by biansy000** [comment]
Is there any solution for more stable upperbody estimation? We are now facing similar issues.

**Comment by penincillin** [comment]
@krigeta @biansy000 
To resolve the un-stability issue caused by occlusion (in your cases, only upperbody visible). You can check those papers focusing on handling occlusions. For example, [3D MULTI-BODIES](https://sites.google.com/view/3dmb/home), which is another paper from Dr. Joo

**Comment by biansy000** [comment]
thanks, the paper is very helpful

**Comment by krigeta** [comment]
> @krigeta @biansy000
> To resolve the un-stability issue caused by occlusion (in your cases, only upperbody visible). You can check those papers focusing on handling occlusions. For example, [3D MULTI-BODIES](https://sites.google.com/view/3dmb/home), which is another paper from Dr. Joo.

this paper seems nice. hope, the code will launch soon!
